source/_integrations/tplink.markdown

```diff
@@ -35,50 +38,87 @@ There is currently support for the following device types within Home Assistant:
 
 {% include integrations/config_flow.md %}
 
-## Supported Devices
-### Plugs
-
-- HS100
-- HS103
-- HS105
-- HS110 (supports consumption sensors)
-- KP105
-- KP115 (supports consumption sensors)
+## TP-Link Kasa Device Overview
 
-### Strip (Multi-Plug)
+### Plugs
 
-- HS107 (indoor 2-outlet)
-- HS300 (powerstrip 6-outlet) (supports consumption sensors)
-- KP303 (powerstrip 3-outlet)
-- KP400 (outdoor 2-outlet)
-- KP200 (indoor 2-outlet)
-- KP40 (outdoor 2-outlet)
-- EP40 (outdoor 2-outlet)
+| Model Number     | Support by Home Assistant | Support by TP-Link's 'local only mode'     | Note                                     |
+|------------------|---------------------------|--------------------------------------------|------------------------------------------|
+| HS100            | ✅                        | ✅                                         |                                          |
+| HS103            | ✅                        | ✅                                         |                                          |
+| HS105            | ✅                        | ✅                                         |                                          |
+| HS110            | ✅                        | ✅                                         | supports consumption sensors             |
+| KP100            | ❓                        | ✅                                         |                                          |
```

Comment:
These work.  I have a few of these.  I'll add fixtures for them to `python-kasa` when I get a chance.

Reply:
https://github.com/python-kasa/python-kasa/pull/343